.circleci/config.yml

```diff
@@ -40,7 +40,7 @@ aliases:
 
   - &create_test_db
     name: Create database
-    command: cd src/api; bundle exec rake db:create db:structure:load db:seed RAILS_ENV=test
+    command: cd src/api; bundle exec rake db:setup RAILS_ENV=test
```

Comment:
`db:setup` does not include create 

https://github.com/rails/rails/blob/master/activerecord/lib/active_record/railties/databases.rake#L340

Reply:
Somebody should tell that to its description which literally says: `Creates the database, (...)`

Reply:
The CI would also not work if the database wasn't created.